client/payment-details/payment-method/index.js

```diff
@@ -11,14 +11,21 @@ import { Card, CardBody, CardHeader } from '@wordpress/components';
  */
 import Loadable from 'components/loadable';
 import CardDetails from './card';
+import CardPresentDetails from './card-present';
 import GiropayDetails from './giropay';
 import SepaDetails from './sepa';
 import SofortDetails from './sofort';
 
 const PaymentDetailsPaymentMethod = ( { charge = {}, isLoading } ) => {
-	let PaymentMethodDetails = CardDetails;
+	let PaymentMethodDetails = <></>;
```

Comment:
🤔 Haven't tested but looks like there is a risk of getting error in case of this flow:
1. Use default value of `<></>`.
2. Do not reassign within switch case. E.g. unhandled details.type.
3. Then `<></>` will be rendered as component with props. Not sure if error will be thrown here, though. But it still concerns me: that in one case the variable contains component (function) in the other case it holds empty jsx.

Reply:
Yup, I did it to allow for graceful degradation in the event another unrecognized payment type materialized. This way the details might be blank, but other components on the page would still render. Right now, before this change, it breaks all rendering for unrecognized payment types

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/1840/commits/c9b58c63b20bfbb86a8f2888960c2dd21e399a46